src/main/java/org/reactome/release/qa/check/HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities.java

```diff
@@ -0,0 +1,83 @@
+package org.reactome.release.qa.check;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.reactome.release.qa.annotations.SliceQATest;
+import org.reactome.release.qa.common.AbstractQACheck;
+import org.reactome.release.qa.common.QACheckerHelper;
+import org.reactome.release.qa.common.QAReport;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Flags human ReactionlikeEvents that do not have a populated disease attribute, but have non-human participants.
+ */
+
+@SliceQATest
+public class HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities extends AbstractQACheck {
+
+    private static final String innateImmunityPathwayDbId = "168249";
```

Comment:
Nice, thanks for adding the named constant.  For my famous PR nit-picking, could the name be upper case with underscores instead of camel case (i.e. `INNATE_IMMUNITY_PATHWAY_DB_ID`)?

Reply:
Another good catch -- I forgot about that for global variables